Lib/test/regrtest.py

```diff
@@ -1910,6 +1938,70 @@ def getexpected(self):
         assert self.isvalid()
         return self.expected
 
+
+if threading is not None:
+    class WatchDogThread(threading.Thread):
+        daemon = True
+
+        def __init__(self, timeout):
+            threading.Thread.__init__(self)
+            self.timeout = timeout
+            self.stream = sys.__stdout__
```

Comment:
Wouldn't `__stderr__` be more appropriate for debug and error messages?

Can't standard streams be `None` on Windows?